notebooks/agentic_rag_pipeline_with_memory_and_chat.ipynb

```diff
@@ -0,0 +1,530 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Gt3tjDtpWX-A"
+      },
+      "source": [
+        "## Agentic RAG Pipeline with memory and chat\n",
+        "\n",
+        "_notebook by Stefano Fiorucci ([Twitter](https://x.com/theanakin87), [LI](https://www.linkedin.com/in/stefano-fiorucci/)), Bilge Yucel ([Twitter](https://x.com/bilgeycl), [LI](https://www.linkedin.com/in/bilge-yucel/)),  and Tilde Thurium ([Twitter](https://x.com/annthurium), [LI](https://www.linkedin.com/in/annthurium/))_\n",
```

Comment:
Stefano made a good point about the authorship. It's not particularly important to me but here it makes sense to list Vladimir and me instead of Stefano.